app/src/main/java/com/keylesspalace/tusky/entity/Status.kt

```diff
@@ -21,6 +21,7 @@ import com.google.gson.annotations.SerializedName
 import com.keylesspalace.tusky.util.parseAsMastodonHtml
 import java.util.Date
 
+/** A Mastodon status as the server sees it. (Distinct from ConversationStatusEntity or StatusViewData.Concrete). */
 data class Status(
```

Comment:
@connyduck : For consistency with the other types this should be called `StatusEntity`, is that right?

@mcclure : I'm **not** suggesting making that rename in this PR, but if it's correct it would be worth adding that as a TODO here, and/or a note that explains why it's not got the same naming scheme.